music21/roman.py

```diff
@@ -898,6 +948,12 @@ def romanNumeralFromChord(chordObj,
         elif rnString in ('Fr43', 'Sw43'):
             keyObj = _getKeyFromCache(chordObj.seventh.name.lower())
 
+    if preferSecondaryDominants and (ft.prefix == ''):  # Nothing on altered scale degrees
```

Comment:
Ah, I see -- we're definitely seeing this two different ways.

In c minor, with `preferSecondaryDominants=True`, I would expect Eb G Bb to be left alone as III and expect En G# Bn to become V/vi.

Now that I see that out loud, I actually wouldn't expect a diminished chord on vi to be tonicized at all, so I would probably drop this swap entirely. Reactions?

Reply:
Indeed! As below. Thanks.